apps/docs/src/examples/core/three-arcade-game/game/scenes/Intro.svelte

```diff
@@ -42,7 +42,7 @@
     showPressSpaceToStart = true
   }, showPressSpaceToStartAfter)
 
-  let intervalHandler = setInterval(() => {
+  let intervalHandler = window.setInterval(() => {
```

Comment:
So these changes happened because the `setInterval` call happened to create an ci/cd check error because it thought the return type wasn't just a number 🤷‍♂️ I guess somewhere i didn't wrangle the types correctly. this is an fix till then.